src/steps/relay.py

```diff
@@ -34,6 +40,27 @@ def setup_main_relay_nodes(self, request):
         self.node2.start(relay="true", discv5_bootstrap_node=self.enr_uri)
         self.main_nodes.extend([self.node1, self.node2])
 
+    @pytest.fixture(scope="function")
+    def register_rln_main_relay_nodes(self, request):
+        logger.debug(f"Registering RLN credentials: {inspect.currentframe().f_code.co_name}")
+        rln_creds_source = "./rln_creds.json"
+        self.node1 = WakuNode(NODE_2, f"node1_{request.cls.test_id}")
```

Comment:
typo? shouldn't we have NODE_1 here?

Reply:
NODE_1 Is go-waku node. Our old tests still work with Go-waku, so I picked NODE_2. Ideally node id is not only image name. Not sure if this PR is good place to resolve this.

Reply:
Oh right, but then we should use DEFAULT_NWAKU instead of NODE_1 and NODE_2

Reply:
Great, I've swapped this value in tests and fixtures. https://github.com/waku-org/waku-interop-tests/pull/16/commits/8840c80fa83afe4607e2ec379ae0c4c5a6cc3d8d